config/routes.rb

```diff
@@ -204,7 +204,7 @@
     end
   end
   match 'mentor_term_groups/term/:term_id' => 'mentor_term_groups#term', :as => :mentor_term_groups_term
-  resources :volunteers, :only => [:show, :background_check_responses]
+  resources :volunteers, :only => [:show, :background_check_responses], :controller => 'mentors'
```

Comment:
Line is too long. [96/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.